spec/lib/jobs/obm-control-spec.js

```diff
@@ -151,6 +182,19 @@ describe("Job.Obm.Node", function () {
                 expect(ObmService.prototype.reboot).to.have.been.calledOnce;
             });
         });
+
+        it('should fail to run OBM command if no target or nodeId was specified', function() {
+
+            // local options with no nodeId field
+            var options = {
+                action: 'reboot',
+                obmServiceName: 'ipmi-obm-service'
+            };
+
+            expect(function() {
+                return new Job(options, {}, uuid.v4());
+            }).to.throw('nodeId (string) is required');
```

Comment:
Luckily, we can have the best of both worlds. When I want to preserve error message flexibility I usually just do a regex to make sure it's at least *probably* the right error message:

```
expect(xxx).to.throw(AssertionError, /nodeId/)
```